stylesheets/layout/_footer.scss

```diff
@@ -4,64 +4,76 @@
 .footer {
     padding-top: 102px;
     padding-bottom: 95px;
-    border-top: 1px solid #BCD0E5;
-}
-
-.footer__wrapper {
-    display: flex;
-    justify-content: flex-start;    
-}
-
-.footer__logo {
-    max-width: 205px;      
-}
-
-.footer__logo-text {
-    padding-top: 5px;
-    padding-left: 5px;
-}
-
-.footer-first {
-    margin-left: 139px;
-}
-
-.footer-second {
-    margin-left: 144px;
-    margin-right: 82px;
-}
-
-.footer__nav {
-    max-width: 205px;
+    border-top: 1px solid $color-btn-sign-in-border;
 
-    ul {
-        list-style: none;
-        padding-left: 0;
+    &__wrapper {
+        display: flex;
+        justify-content: flex-start;    
     }
-
-    li {
-        font-size: 16px;
-        line-height: 35px;
+    
+    &__logo {
+        max-width: 205px;      
+    }
+    
+    &__logo-text {
+        padding-top: 5px;
+        padding-left: 5px;
+    }
+    
+    &-first {
+        margin-left: 139px;
+    }
+    
+    &-second {
+        margin-left: 140px;
+        margin-right: 82px;
     }
 
-    .nav__title {
-        font-family: $font-family-footer;
-        font-style: normal;
-        font-weight: 900;
-        font-size: 20px;
-        line-height: 30px;
-        color: $color-text;
-        margin-top: 0;
-        margin-bottom: 35px;
+    &-link:hover{
+        text-decoration: underline;        
+    }
+    
+    &__nav {
+        max-width: 205px;
+    
+        ul {
+            list-style: none;
+            padding-left: 0;
+        }
```

Comment:
This class can be added to common styles and can be re-used

Reply:
completed